go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/opencontainers/runc
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.0
```

Comment:
This line should not be needed

Reply:
This is `go get` fiddling with things. There's a discussion at https://github.com/golang/go/issues/65847, other people seem to be complaining. I will work around that.

Reply:
@AkihiroSuda so, once I do that, [this check](https://github.com/opencontainers/runc/actions/runs/10799570842/job/29955598612?pr=4397) fails. If I do run `go mod tidy`, it will end up readding `go 1.22.0` and the toolchain line. Thoughts?

Reply:
So, this is the issue:
```
go: golang.org/x/exp@v0.0.0-20240909161429-701f63a606c0 requires go@1.22.0, but 1.22 is requested
```

Reply:
toolchain line removed.